src/lib/firebase/client/firestore/musicians.ts

```diff
@@ -48,11 +49,13 @@ export async function GetMusisciansThatPlaySongs(ids: string[]) {
 		const playsInSongData = playsInSong.data()
 
 		// Add musician
-		musiciansForSongs[playsInSongData.song.id].push({
+		const musician = {
```

Comment:
```suggestion
		const musician : Musician = {
```

Reply:
Is het uberhaupt nodig om dit te typen trouwens? Feels like dat typescript dit impliciet zou moeten snappen

Reply:
Ja nee is idd overbodig